djhtml/tests/test_suite.py

```diff
@@ -0,0 +1,34 @@
+import os
```

Comment:
Please don't place tests under the package. This way the test suite will be added to the deployed package.